tox.ini

```diff
@@ -9,6 +9,9 @@ commands =
 deps = -rdev-requirements.txt
 install_command = pip install --pre {opts} {packages}
 
+[testenv:functional]
+commands = xml2kinto -s http://localhost:8888/v1 -x blocklist.xml -S schemas.json
```

Comment:
Usually I try to avoid having tox to call Make because they do not necessary use the same virtualenv. I would rather have ``make functional`` call ``tox -e functional`` what do you think?